src/barkhaPJun20/StaticNonstatic.java

```diff
@@ -0,0 +1,46 @@
+package barkhaPJun20;
+
+public class StaticNonstatic {
+	 double sum( double x, double y, double z){
+		return (x+y+z);
+	}	 
+	  double average(double x1, double y1,double z1){
+		StaticNonstatic s2=new StaticNonstatic();
```

Comment:
Non static to non static method.. object is not required. please call directly.

Reply:
Suggested changes are done